docs/resources/User.md

```diff
@@ -126,7 +126,7 @@ The connection object that the user has attached.
 | xbox               | Xbox                |
 | youtube            | YouTube             |
 
-\* Users can no longer connect to these services, but the connections weren't removed from the users that already had it.
+\* Users can no longer connect acccounts from these services, but the connections weren't removed from the users that had them.
```

Comment:
```suggestion
\* Users can no longer connect accounts from these services, but the connections weren't removed from the users that had them.
```

Reply:
why is this commented even needed, bots don't need to know if users can still obtain them

Reply:
to avoid confusion from users when they see old connections and wonder why they're not in the ui